anime-list-master.xml

```diff
@@ -28562,8 +28563,8 @@
   <anime anidbid="10384" tvdbid="79214" defaulttvdbseason="2" episodeoffset="" tmdbid="" imdbid="">
     <name>Mushishi Zoku Shou</name>
     <mapping-list>
-      <mapping anidbseason="0" tvdbseason="0">;1-2;2-3;</mapping>
-      <mapping anidbseason="1" tvdbseason="0">;11-4+5;</mapping>
+      <mapping anidbseason="0" tvdbseason="0">;1-0;2-0;</mapping>
+      <mapping anidbseason="1" tvdbseason="0">;11-2+3;</mapping>
```

Comment:
Does this actually work?! 😮
I should test it and add it to the readme

Reply:
It does in my Plex setup! I am not sure if it is going to work for other titles or other scanners though. More testing is required.

Have a look at:

```txt
Mushi-Shi The Next Passage - E11 - Path of Thorns.mkv
```

<img width="826" alt="Screen Shot 2022-09-22 at 9 40 05 AM" src="https://user-images.githubusercontent.com/5350216/191675947-995d7353-aafc-4de6-9dd8-9cbf7807cf4d.png">